airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/registry_entry.py

```diff
@@ -578,14 +600,55 @@ def metadata_entry(context: OpExecutionContext) -> Output[Optional[LatestMetadat
     return Output(value=metadata_entry, metadata=dagster_metadata)
 
 
+def find_release_candidates_for_metadata_entry(
```

Comment:
Looks like this isn't used?

Reply:
Thanks for catching this. It's not used since my latest refactoring.